app/models/batch_context.rb

```diff
@@ -45,13 +51,17 @@ class BatchContext < ApplicationRecord
   # return [PreAssembly::Batch]
   def batch
     @batch ||= if using_file_manifest
-                 manifest = PreAssembly::FileManifest.new(csv_filename: file_manifest, staging_location: staging_location)
-                 PreAssembly::Batch.new(self, file_manifest: manifest)
+                 PreAssembly::Batch.new(self, file_manifest: file_manifest)
                else
                  PreAssembly::Batch.new(self)
                end
   end
 
+  def file_manifest
+    PreAssembly::FileManifest.new(csv_filename: file_manifest_path,
```

Comment:
not in love with "csv_filename" either.

Reply:
Me either, but this is what was here before.